src/bots/depositor.py

```diff
@@ -128,55 +129,29 @@ def execute(self, block: BlockData) -> bool:
 
         return True
 
-    def _is_mellow_depositable(self, module_id: int) -> bool:
-        if not variables.MELLOW_CONTRACT_ADDRESS:
-            return False
-        try:
-            buffered = self.w3.lido.lido.get_buffered_ether()
-            unfinalized = self.w3.lido.lido_locator.withdrawal_queue_contract.unfinalized_st_eth()
-            if buffered < unfinalized:
-                return False
-            staking_module_contract: StakingModuleContract = self.w3.lido.simple_dvt_staking_strategy.staking_module_contract
-            if staking_module_contract.get_staking_module_id() != module_id:
-                logger.debug(
-                    {
-                        'msg': 'Mellow module check failed.',
-                        'contract_module': staking_module_contract.get_staking_module_id(),
-                        'tx_module': module_id,
-                    }
-                )
-                return False
-            balance = self.w3.lido.simple_dvt_staking_strategy.vault_balance()
-        except Exception as e:
-            logger.warning(
-                {
-                    'msg': 'Failed to check if mellow depositable.',
-                    'module_id': module_id,
-                    'err': repr(e),
-                }
-            )
-            return False
-        MELLOW_VAULT_BALANCE.labels(module_id).set(balance)
-        if balance < variables.VAULT_DIRECT_DEPOSIT_THRESHOLD:
-            logger.info(
-                {
-                    'msg': f'{balance} is less than VAULT_DIRECT_DEPOSIT_THRESHOLD while building mellow transaction.',
-                    'balance': balance,
-                    'threshold': variables.VAULT_DIRECT_DEPOSIT_THRESHOLD,
-                }
-            )
-            return False
-        logger.debug({'msg': 'Mellow module check succeeded.', 'tx_module': module_id})
-        return True
-
     def _check_balance(self):
+        eth_chain_id = self.w3.eth.chain_id
+
         if variables.ACCOUNT:
             balance = self.w3.eth.get_balance(variables.ACCOUNT.address)
-            ACCOUNT_BALANCE.set(balance)
+            ACCOUNT_BALANCE.labels(variables.ACCOUNT.address, eth_chain_id).set(balance)
             logger.info({'msg': 'Check account balance.', 'value': balance})
-        else:
-            logger.info({'msg': 'No account provided. Dry mode.'})
-            ACCOUNT_BALANCE.set(0)
+
+        logger.info({'msg': 'Check guardians balances.'})
+
+        w3_databus, w3_databus_chain_id = None, None
+        if variables.ONCHAIN_TRANSPORT_RPC_ENDPOINTS:
+            w3_databus = Web3(MultiProvider(variables.ONCHAIN_TRANSPORT_RPC_ENDPOINTS))
+            w3_databus_chain_id = w3_databus.eth.chain_id
+
+        guardians = self.w3.lido.deposit_security_module.get_guardians()
+        for address in guardians:
+            eth_balance = self.w3.eth.get_balance(address)
+            GUARDIAN_BALANCE.labels(address=address, chain_id=eth_chain_id).set(eth_balance)
```

Comment:
This means that the metric will be updated only for depositor. In a situation when the depositor become unavailable, but other bots are working we will loose the metric. Is it intended?